### PR TITLE
Question Answering for TF trainer

### PR DESCRIPTION
**Pull request by jplu**
This PR add the Question Answering task in the Tensorflow trainer. For the moment the evaluation is not possible due to the complexity of the SQuAD metrics but the training is working.

### CODE REVIEW

File: src/transformers/training_args_tf.py [hunk 1]
```diff
@@ -22,7 +22,9 @@ class TFTrainingArguments(TrainingArguments):
     )
     mode: str = field(
         default="sequence-classification",
-        metadata={"help": 'Type of task, one of "sequence-classification", "token-classification" '},
+        metadata={
+            "help": 'Type of task, one of "sequence-classification", "token-classification", "question-answering"'
```
Comment (julien-c) [inline src/transformers/training_args_tf.py:26]: (not related to this PR) 

we should probably change "sequence-classification" to "text-classification" to be consistent with:

- pipelines
- labels on the website

**Review by julien-c — APPROVED** [review]
Awesome 💪

**Comment by julien-c** [comment]
LMK when it's ready to merge

File: src/transformers/training_args_tf.py (hunk 1 above)
Comment (jplu) [reply src/transformers/training_args_tf.py]: Done!

**Comment by jplu** [comment]
Good to merge!

**Review by LysandreJik — APPROVED** [review]
Very nice!

**Pull request merged**